### PR TITLE
Add mtu functional test

### PR DESCRIPTION
**Pull request by SchSeba**
Add test for both netdevice and for vfio

This PR also update the OCP version to 4.16, this is needed to have the newer multus-cni that support the MTU network-status field

### CODE REVIEW

File: test/conformance/tests/test_sriov_operator.go [hunk 1]
```diff
@@ -1157,6 +1157,39 @@ var _ = Describe("[sriov] operator", func() {
...
+						},
+						Spec: sriovv1.SriovNetworkSpec{
+							ResourceName:     resourceName,
+							IPAM:             `{"type":"host-local","subnet":"10.10.10.0/24","rangeStart":"10.10.10.171","rangeEnd":"10.10.10.181","routes":[{"dst":"0.0.0.0/0"}],"gateway":"10.10.10.1"}`,
+							NetworkNamespace: namespaces.Test,
+						}}
+
+					err = clients.Create(context.Background(), sriovNetwork)
+					Expect(err).ToNot(HaveOccurred())
+					Eventually(func() error {
```
Comment (zeeke) [inline test/conformance/tests/test_sriov_operator.go:1175]: can you use `waitForNetAttachDef()`?

https://github.com/k8snetworkplumbingwg/sriov-network-operator/blob/ee2ee7422b0ca4ea811e242f1a5d2fcc012a36fd/test/conformance/tests/test_sriov_operator.go#L2819

**Review by zeeke — COMMENTED** [review]
Added one minor comment. Other than that, LGTM

File: test/conformance/tests/test_sriov_operator.go (hunk 1 above)
Comment (SchSeba) [reply test/conformance/tests/test_sriov_operator.go]: you are right!

done

**Review by adrianchiris — APPROVED** [review]

**Review by zeeke — APPROVED** [review]
LGTM

**Pull request merged**